test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1362,23 +1387,414 @@ static public void testGCFlattenedMegaObjectArray() throws Throwable {
+		assertFalse((double1 == negativeInfinity), "A substitutability (==) test on different value should always return false");
+		assertFalse((positiveInfinity == negativeInfinity), "A substitutability (==) test on different value should always return false");
+		assertFalse((positiveZero == negativeZero), "A substitutability (==) test on different value should always return false");
+		assertFalse((double1 != double1), "A substitutability (!=) test on the same value should always return false");
+		assertFalse((double1 != double4sameAs1), "A substitutability (!=) test on different value the same contents should always return false");
+		assertFalse((nan != nan2), "A substitutability (!=) test on different value the same contents should always return false");
+		assertTrue((double1 != double2), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != double3), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != nan), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != positiveZero), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != negativeZero), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != positiveInfinity), "A substitutability (!=) test on different value should always return true");
+		assertTrue((double1 != negativeInfinity), "A substitutability (!=) test on different value should always return true");
+		assertTrue((positiveInfinity != negativeInfinity), "A substitutability (!=) test on different value should always return true");
+		assertTrue((positiveZero != negativeZero), "A substitutability (!=) test on different value should always return true");
+	}
+
+	@Test(priority=6)
+	static public void testACMPTestOnAssortedValues() throws Throwable {
+		Object assortedValueWithLongAlignment = AssortedValueWithLongAlignment.createObjectWithDefaults();
+		Object assortedValueWithLongAlignment2 = AssortedValueWithLongAlignment.createObjectWithDefaults();
+		Object assortedValueWithLongAlignment3 = AssortedValueWithLongAlignment.createObjectWithDefaults();
+		assortedValueWithLongAlignment3 = ((AssortedValueWithLongAlignment)assortedValueWithLongAlignment3).checkFieldsWithDefaults();
+
+		Object assortedValueWithObjectAlignment = AssortedValueWithObjectAlignment.createObjectWithDefaults();
+		Object assortedValueWithObjectAlignment2 = AssortedValueWithObjectAlignment.createObjectWithDefaults();
+		Object assortedValueWithObjectAlignment3 = AssortedValueWithObjectAlignment.createObjectWithDefaults();
+		assortedValueWithObjectAlignment3 = ((AssortedValueWithObjectAlignment)assortedValueWithObjectAlignment3).checkFieldsWithDefaults();
+
+		// TODO add commented out tests once withfield is replaced
```

Comment:
Instead of commenting out the tests, you can add a variable like `withfieldRemoved`, and set it to false. Then put the commented tests inside `if (withfieldRemoved) `

Reply:
Yes, you can do them together.

Reply:
Okay. Do you mind if I do this in a seperate pr? it will make this change quite a bit bigger

Reply:
It can be done in a separate PR.

Reply:
Okay. I think this is ready then unless you have any other comments.

Reply:
> Okay. I think this is ready then unless you have any other comments.

I am waiting for you to remove this https://github.com/eclipse-openj9/openj9/pull/18415#discussion_r1388166115

Reply:
updated.